src/main/resources/add-ons-to-index.json

```diff
@@ -3852,6 +3897,7 @@
       "uid": "org.openmrs.module.registration-core-module",
       "type": "OMOD",
       "name": "Registration Core Module",
+      "icon": "users",
```

Comment:
This one isn't really relevant so I guess this should be replaced/ deleted. User-plus would fit better